Dockerfile

```diff
@@ -33,7 +33,8 @@ RUN set -x \
  && /fossology/utils/fo-installdeps -e -y \
  && $_cleanup
 RUN curl -sS https://getcomposer.org/installer | php && \
-    mv composer.phar /usr/local/bin/composer
+	mkdir /tmp/composer && \
+    mv composer.phar /tmp/composer/composer
```

Comment:
**Idea:** on Projectbuilder setup install composer to `/tmp/composer/composer` and add `/tmp/composer` to `$PATH` before calling `make phpvendors` resp `make composer_install` and remove all changes related to the placement of composer.